problems/sumDigits.yaml

```diff
@@ -0,0 +1,40 @@
+id: 3
+title: SumDigits
+text: |
+  Write a function called sumDigits that takes in an integer,
+  and returns that sum of all its digits. If the number
+  is negative, the first digit should count as the only negative.
+  
+  Refer to example below.
+  
+  sumDigits(123456) = 1 + 2 + 3 + 4 + 5 + 6 = 21
+  sumDigits(-123456) = -1 + 2 + 3 + 4 + 5 + 6 = 19
+code: |
+  function sumDigits(num) {
+    return num;
+  }
+
+  assert(sumDigits(0) === 0);
+  assert(sumDigits(123) === 6);
+  assert(sumDigits(456) === 15);
+  assert(sumDigits(789) === 24);
+  assert(sumDigits(-123) === 0);
+  assert(sumDigits(-456) === 0);
+  alert('Passed all test cases!');
+tests:
+  - assert(sumDigits(0) === 0);
+  - assert(sumDigits(123) === 6);
+  - assert(sumDigits(456) === 15);
+  - assert(sumDigits(789) === 24);
+  - assert(sumDigits(-123) === 0);
+  - assert(sumDigits(-456) === 0);
+solution: |
+  function sumDigits(num) {
```

Comment:
this seems to fail on negative numbers

Reply:
I'm sorry. I forgot to change those .

Reply:
i meant to say the code itself will not work on negative numbers.
sumDigits(-123) will give -6 instead of 4.

Reply:
Ohhh I see !